### ISSUE TITLE
Therapeutics Data Commons

### ISSUE THREAD

**Issue by rbharath** [deepchem/moleculenet#33]
A really nice paper just came out that curates 66 therapeutically relevant datasets: https://arxiv.org/abs/2102.09548. They have documentation and an API up at https://tdcommons.ai/.

**Comment by yuanqidu** [comment]
Should we consider to expand this benchmark by more methods and/or datasets for moleculenet V2.0?

**Comment by rbharath** [comment]
Yes definitely! I believe `tdc` has a python API. Perhaps we could write a wrapper to expose `tdc` datasets as part of MoleculeNet?
